src/udev/udev-ctrl.c

```diff
@@ -332,13 +331,6 @@ int udev_ctrl_send(UdevCtrl *uctrl, UdevCtrlMessageType type, const void *data)
         return 0;
 }
 
-int udev_ctrl_send_ping(UdevCtrl *uctrl) {
-        assert(uctrl);
-        assert(uctrl->varlink);
-
-        return udev_varlink_call(uctrl->varlink, "io.systemd.udev.Ping", NULL, NULL);
-}
```

Comment:
hmm, you are dropping something you reworked majorly in an earlier commit in the same PR. maybe drop the function right-away?

Reply:
Yeah, that rework you mention was done so I could convert the simplest command (`ping`) first. Now that I know the whole thing works, I can rearrange the commits to avoid this addition and subsequent removal of `uctrl->varlink` member. I just forgot to do it during the latest rebase.